src/engine/ui/RmlUi_Platform_GLFW.h

```diff
@@ -32,6 +32,7 @@
 #include <RmlUi/Core/SystemInterface.h>
 #include <RmlUi/Core/Types.h>
 
+namespace RmlGLFW {
```

Comment:
This does not warrant its own namespace, make it part of `cqsp::engine::rml` or something like that.